src/compose/ComposeBox.js

```diff
@@ -259,6 +259,16 @@ export default class ComposeBox extends PureComponent<Props, State> {
     const messageHeight = Math.min(Math.max(MIN_HEIGHT, height + 12), MAX_HEIGHT);
     const totalHeight = canSelectTopic ? messageHeight + 30 : messageHeight;
     const placeholder = getComposeInputPlaceholder(narrow, auth.email, users);
+    const msgInputStyles = {
+      height: messageHeight === MIN_HEIGHT ? MIN_HEIGHT - 12 : messageHeight,
+      ...Platform.select({
+        android: {
+          paddingTop: messageHeight === MAX_HEIGHT ? 6 : 0,
```

Comment:
Actually applying `padding: 0` fixes it. (On Android input have its on default padding.)
But when it becomes scrollable on Android, it default padding vanishes and doesn't looks good. So applying padding when we reach `MAX_HEIGHT`.